### DISCUSSION TITLE
i met some installation promblems

### DISCUSSION THREAD

**Discussion by hpz1994** [Electrostatics/apbs#206] [Q&A]
Dear all,
I'm trying to install on CentOS 7 with .build.sh but stoped with an error:
Configuring build for MC
-- FETK set_basic_vars_and_paths: FALSE, mc
CMake Error at /usr/share/cmake3/Modules/FindPackageHandleStandardArgs.cmake:                                                                                164 (message):
  Could NOT find UMFPACK (missing: UMFPACK_INCLUDES UMFPACK_LIBRARIES
  UMFPACK_LIBDIR)
Call Stack (most recent call first):
  /usr/share/cmake3/Modules/FindPackageHandleStandardArgs.cmake:445 (_FPHSA_F                                                                                AILURE_MESSAGE)
  build/_deps/fetk-src/cmake/FindUMFPACK.cmake:36 (find_package_handle_standa                                                                                rd_args)
  build/_deps/fetk-src/mc/CMakeLists.txt:37 (find_package)
Does anyone know how to fix this?

**Comment by nsoblath** [comment]
Hi @hpz1994.  It looks like you're missing one of the dependencies needed to build APBS.  You can install UMFPACK as part of the SuiteSparse package.  I haven't tried a build on CentOS 7, but I believe the package `suitesparse-devel` will give you UMFPACK.
